gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -385,8 +389,8 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
       IGN_PROFILE_BEGIN("fill ROS message");
     #endif
 
-      // Populate message
-      auto slip_msg = gazebo_msgs::msg::WheelSlip();
+      // Reset message
+      slip_msg = gazebo_msgs::msg::WheelSlip();
```

Comment:
```suggestion
      impl_->slip_msg = gazebo_msgs::msg::WheelSlip();
```

Reply:
it won't let me comment on the lines below, but all occurrences of `slip_msg` need to be `impl_->slip_msg`